rules/macos/credential_access_credentials_keychains.toml

```diff
@@ -67,22 +67,29 @@ process where host.os.type == "macos" and event.type in ("start", "process_start
       "login.keychain"
     ) and
     not process.args : ("find-certificate",
-                      "add-trusted-cert",
-                      "set-keychain-settings",
-                      "delete-certificate",
-                      "/Users/*/Library/Keychains/openvpn.keychain-db",
-                      "show-keychain-info",
-                      "lock-keychain",
-                      "set-key-partition-list",
-                      "import",
-                      "find-identity") and
+                        "add-trusted-cert",
+                        "set-keychain-settings",
+                        "delete-certificate",
+                        "/Users/*/Library/Keychains/openvpn.keychain-db",
+                        "show-keychain-info",
+                        "lock-keychain",
+                        "set-key-partition-list",
+                        "import",
+                        "find-identity") and
     not process.parent.executable :
       (
         "/Applications/OpenVPN Connect/OpenVPN Connect.app/Contents/MacOS/OpenVPN Connect",
         "/Applications/Microsoft Defender.app/Contents/MacOS/wdavdaemon_enterprise.app/Contents/MacOS/wdavdaemon_enterprise",
         "/opt/jc/bin/jumpcloud-agent"
       ) and
-    not process.executable : "/opt/jc/bin/jumpcloud-agent"
+    not process.executable : "/opt/jc/bin/jumpcloud-agent" and
```

Comment:
Do we want to specifically ignore `/usr/bin/basename` from telemetry? I noticed, similar to JAMF, that there are quite a bit of FPs generated from this executable it would seem?